src/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Leaderboard</title>
+</head>
+<body>
+	<h1 class="" id="title">Leaderboard</h1>
+
+	<main class="" id="main-UI">
+		
+		<div class="" id="output-UI">
+			
+			<header id="oui-header">
+				<h2 class="" id="oui-title">Recent scores</h2>
+				<button id="refresh-btn" type="button">Refresh</button>
+			</header>
+
+			<div class="" id="oui-scores">
+				<!-- Content created dynamically -->
+			</div>
+
+		</div>
+
+		<div class="" id="input-UI">
+
+			<h2 class="" id="iui-title">Add your score</h2>
+
+			<input id="iui-name" type="text">
+			<input id="iui-score" type="number">
+
+			<button id="submit-btn" type="button">Submit</button>
+
+		</div>
+	
```

Comment:
- [ ] it would be better if you follow the same wireframe provided and add some space between recent scores and add a score and also the button refresh should be in the right of the recent scores header so please try to implement that 
- wireframe 
<img width="622" alt="Screen Shot 2022-10-13 at 15 43 47" src="https://user-images.githubusercontent.com/61790587/195629811-ed4eda1e-bad6-446c-9727-546403aa5777.png">
- Yours 
<img width="884" alt="Screen Shot 2022-10-13 at 15 43 08" src="https://user-images.githubusercontent.com/61790587/195629863-57da336b-9e2e-4b7b-ae2b-ae39ca5eab4a.png">

Reply:
Thanks @youmari I've updated the design.